Misc/NEWS.d/next/Core and Builtins/2024-02-07-00-18-42.gh-issue-112069.jRDRR5.rst

```diff
@@ -0,0 +1 @@
+Convert set and frozenset methods to Argument Clinic
```

Comment:
I'm not sure if this change needs a news entry, but if it does, we should use the RST markups here.

```suggestion
Convert :class:`set` and :class:`frozenset` methods to Argument Clinic.
```

Reply:
Dunno, for now I updated it per your suggestion ;)